html/README.md

```diff
@@ -164,4 +164,17 @@ Don't use lists as container for a lot of data, only for simple lists.
         </div>
     </li>
 </ul>
-```
\ No newline at end of file
+```
+
+## Use the <button> element for buttons
+Don't use the input element for this.
```

Comment:
Please explain why this is

Reply:
I think we need to add a description like: *"Don't use the input element for buttons since they're 
 way more flexible because of there HTML support."*. And maybe move this convention next to the [button type convention](https://github.com/AanZee/conventions/blob/master/html/README.md#add-type-to-button-elements).

Reply:
I think we need to add a description like: *"Don't use the input element for buttons since they're 
 way more flexible because of there HTML support."*. And maybe move this convention next to the [button type convention](https://github.com/AanZee/conventions/blob/master/html/README.md#add-type-to-button-elements).